README.md

```diff
@@ -85,7 +186,7 @@ This plugin can be consumed by the CAP application deployed on BTP to store thei
    <dependency>
       <groupId>com.sap.cds</groupId>
       <artifactId>sdm</artifactId>
-      <version>1.0.0-SNAPSHOT</version>
+      <version>RELEASE</version>
    </dependency>
```

Comment:
I think this should be the first step. Otherwise, Attachments aspect will thrown compilation error in cds file.

Reply:
Also just say "Add the following in dependency in pom.xml in the _srv_ folder"

Reply:
Done